include/ginkgo/core/base/executor.hpp

```diff
@@ -1040,6 +1086,9 @@ class ReferenceExecutor : public OmpExecutor {
     GKO_DEFAULT_OVERRIDE_VERIFY_MEMORY(CudaExecutor, false);
 
     GKO_DEFAULT_OVERRIDE_VERIFY_MEMORY(HipExecutor, false);
+
+private:
+    exec_info ref_exec_info_;
```

Comment:
Now that it's implemented using the same structure for everyone, you could make this a member of the Executor base class and add a public method which returns a reference/pointer to a const version of that structure, and a protected non const getter which you could use to set the parameters.

Reply:
Good point. I have done this now.